modules/ding_frontend/ding_frontend.features.inc

```diff
@@ -898,207 +898,38 @@ function ding_frontend_default_search_api_index() {
       "cron_limit" : "50",
       "fields" : {
         "item_bundle" : { "type" : "string" },
+        "item_entity_id" : { "type" : "string" },
         "item_id" : { "type" : "string" },
+        "item_label" : { "type" : "text" },
         "item_type" : { "type" : "string" },
         "node" : { "type" : "integer", "entity_type" : "node" },
         "node:author" : { "type" : "integer", "entity_type" : "user" },
         "node:changed" : { "type" : "date" },
+        "node:comment" : { "type" : "integer" },
         "node:created" : { "type" : "date" },
-        "node:field_ding_event_category" : { "type" : "integer", "entity_type" : "taxonomy_term" },
```

Comment:
This seems to removes a lot of the search API configuration dealing with libraries and later on users from issue 430 by @cableman.

Why is this removed? That does not seem to be related to P2?